MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -11,6 +11,11 @@ open class RouteProgress: NSObject {
      Returns the current `Route`.
      */
     @objc public let route: Route
+    
+    /**
+     An optional alternate route the user could use for navigation.
+     */
+    @objc public var alternateRoute: Route?
```

Comment:
This is about the least intrusive change I could think of. Making `RouteProgress.route` -> `RouteProgress.routes` would be a pretty huge change.

Reply:
I think we should make that change for consistency with other parts of the API. `route` can still be a computed property serving as an alias for the first route.

Reply:
Yeah the only issue is that init changes to `routes` which becomes a huge headache. Do we then rely on order of `routes` for the active route?

Reply:
Yes, `NavigationMapView` already does that:

https://github.com/mapbox/mapbox-navigation-ios/blob/b57c8e78355211d59e06ce67714298012b129994/MapboxNavigation/NavigationMapView.swift#L89
https://github.com/mapbox/mapbox-navigation-ios/blob/b57c8e78355211d59e06ce67714298012b129994/MapboxNavigation/NavigationMapView.swift#L423